server.js

```diff
@@ -0,0 +1,25 @@
+const express = require("express");
+const app = express();
+const cors = require("cors");
+const PORT = 4000;
+
+app.use(cors());
+app.use(express.json());
+
+const shopItemRouter = require("./routes/shopItem.js");
+app.use("/shopItem", shopItemRouter);
+
+const findIdRouter = require("./routes/findId.js");
+app.use("/shopItem/:id", findIdRouter);
```

Comment:
Maybe something like `shopItemById` would be better since we plan to add other kind of "items" (e.g. recipe) to the server later.